crates/client/rpc-core/src/utils.rs

```diff
@@ -61,3 +61,38 @@ pub(crate) fn _to_deprecated_entrypoint_by_type(
     });
     DeprecatedEntryPointsByType { constructor, external, l_1_handler }
 }
+
+/// Returns a hash map of entry point types to entrypoint from deprecated entry point by type
+pub fn to_btree_map_entrypoints(
+    entries: DeprecatedEntryPointsByType,
+) -> BTreeMap<EntryPointTypeWrapper, BoundedVec<EntryPointWrapper, MaxEntryPoints>> {
+    let mut entry_points_by_type: BTreeMap<EntryPointTypeWrapper, BoundedVec<EntryPointWrapper, MaxEntryPoints>> =
+        BTreeMap::new();
+    // We can unwrap safely as we already checked the length of the vectors
```

Comment:
where is this checked performed ? are you referring to a check outside of the function ? if yes, why do you assume the check will always be done, since the function is public ?